packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -153,6 +154,62 @@ export default {
       type: Object,
       default: null,
     },
+
+    /**
+     * Whether the input allows for block quote.
+     */
+    allowBlockquote: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for bold to be introduced in the text.
+     */
+    allowBold: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for bullet list to be introduced in the text.
+    */
+    allowBulletList: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for italic to be introduced in the text.
+     */
+    allowItalic: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for list item to be introduced in the text.
+     */
+    allowListItem: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for strike to be introduced in the text.
+     */
+    allowStrike: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for underline to be introduced in the text.
+     */
+    allowUnderline: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
There is no way to actually enter an underline in markdown, so it's impossible to set this without a button or similar I guess maybe if you pasted text that contained underline? Also the reason markdown is like this is because underlining text can be confused with links on the web. Maybe we don't need to include this one at all?

Reply:
For me is fine to remove this one, as we are not going to use it. @emariotti3 do you have any concern with this? can we remove it?

Reply:
fyi I think it can be triggered by cmd + shift + u

Reply:
Yeah looks like it.. The editor component actually has a button for it in the UI and it might make more sense in the context of the editor component, so maybe it's best to just keep it. Not needed in the message input though.